lib/android/src/main/java/net/skyscanner/backpack/reactnative/BackpackReactNative.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.reactnative
+
+import android.content.Context
+import android.content.res.Resources
+import android.util.Log
+import com.facebook.react.views.text.ReactFontManager
+
+/**
+ * [BackpackReactNative] is the entry point of backpack-react-native for Android.
+ */
+class BackpackReactNative {
+  companion object {
+    private const val TAG = "BackpackReactNative"
+    private val RELATIVE_MAPPINGS = mapOf(
```

Comment:
Keys are the font names that will be available to RN and values are the font names available in the backpack-android package